packages/redux/src/raffles/actions/factories/fetchRafflesFactory.ts

```diff
@@ -0,0 +1,60 @@
+import * as actionTypes from '../../actionTypes';
+import {
+  Config,
+  GetRaffles,
+  Raffles,
+  toBlackoutError,
+} from '@farfetch/blackout-client';
+import { buildQueryStringFromObject as generateRafflesHash } from '../../../helpers';
+import { normalize } from 'normalizr';
+import raffleSchema from '../../../entities/schemas/raffles';
+import type { Dispatch } from 'redux';
+import type { Nullable } from '../../../types';
+import type { RafflesQuery } from '@farfetch/blackout-client/src/raffles/types/rafflesQuery.types';
```

Comment:
Import it from the root of the package please. (You can update the existing import with that).
Tip: If you use VSCode, write the type you need and it will suggest the import for you.
If did not use the suggestion (for example, you copy/pasted the type from another file) move the cursor on top of the type and press Cmd + .  and VSCode will offer the import suggestions for you and you can choose the correct one.

Reply:
done, pls double check